src/FiatConnectClient.tsx

```diff
@@ -15,9 +19,82 @@ import {
 } from '@fiatconnect/fiatconnect-sdk'
 import { createSiweConfig } from '@fiatconnect/fiatconnect-sdk/dist/fiat-connect-client'
 import { fiatConnectNetworkToChainId } from './constants'
-import { ProviderIds } from './types'
+import { ProviderIds, AddKycParams } from './types'
 
-export function addFiatAccount(
+export async function addKyc<T extends KycSchema>(
+  params: AddKycParams<T>,
+  clientConfig: FiatConnectClientConfig,
+): Promise<KycStatus> {
+  const addKycResponse = await fetch(
+    `${clientConfig.baseUrl}/kyc/${params.kycSchemaName}`,
+    {
+      method: 'POST',
+      credentials: 'include',
+      headers: {
+        'Content-Type': 'application/json',
+        Authorization: `Bearer ${clientConfig.apiKey}`,
+      },
+      body: JSON.stringify(params.data),
+    },
+  )
+  if (!addKycResponse.ok) {
```

Comment:
are you sure 404s have ok set to true?

Reply:
looks like ok is 200-299 only https://fetch.spec.whatwg.org/#statuses

Reply:
looks like you check for 404s in other queries but not this one

Reply:
Namely because of [how this endpoint works](https://github.com/fiatconnect/specification/blob/main/fiatconnect-api.md#3421-post-kyckycschema), a 404 isn't an error state of this endpoint like it is for the other KYC methods - we definitely want to fail if we get a 404 when POSTing here.